package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "fathom-product",
+  "description": "Product extraction using Fathom.",
+  "version": "2.1.0",
```

Comment:
Probably want to start from 1.0 or 0.1 or something. node.js libs use [semantic versioning](http://semver.org/), so I recommend 0.1 until the API stabilizes.